### DISCUSSION TITLE
[Vote ended 2022-12-12] Postpone Ansible 7.2.0

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#169] [Votes done]
Since the ansible-core team announced [they will skip the Dec 26/Jan 2 window](https://groups.google.com/g/ansible-devel/c/IQ7VPnw9yS8) due to the holiday season and the next releases will be at the end of January 2023, I suggest to do the same for the community package.

Please vote with +1 to wait with Ansible 7.2.0 until ansible-core 2.14.2 is released end of January 2023, or -1 to release it earlier.

(Use #170 for discussion *about* this vote.)

**Comment by mariolenz** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by gotmax23** [comment]
Weak +1

**Comment by rooftopcellist** [comment]
+1

**Comment by oraNod** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by cybette** [comment]
+1
